### PR TITLE
[Android] Add ERNIE Tiny android support

### PR DESCRIPTION
**Pull request by DefTruth**
### PR types

New features

### PR changes

Others

### Description
 
  
- Add ERNIE Tiny android support
  - [x] Init ERNIE Tiny JNI template
  - [x] Bind ERNIE Tiny via JNI & support Java API
  - [x] Add simple Android app example for ERNIE Tiny
  - [x] Add ERNIE Tiny Android README.md
  - [x] Add added_tokens_path param

### CODE REVIEW

File: model_zoo/ernie-tiny/deploy/android/README.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+}
+```
+
+### ERNIE Tiny Java API说明如下  
+
+- ERNIE Tiny `Predictor`初始化 API: 模型初始化API包含两种方式，方式一是通过构造函数直接初始化；方式二是，通过调用init函数，在合适的程序节点进行初始化。ERNIE Tiny Predictor初始化参数说明如下：
+   - modelFile: String, paddle格式的模型文件路径，如 infer_model.pdmodel
+   - paramFile: String, paddle格式的参数文件路径，如 infer_model.pdiparams
+   - vocabFile: String, 词表文件，如 vocab.txt 每一行包含一个词
+   - slotLabelsFile: String, 槽位标签文件，如 slots_label.txt 每一行包含一个标签
```
Comment (joey12300) [inline model_zoo/ernie-tiny/deploy/android/README.md]: 添加一个`added_tokens_path`参数，可仿照我的pr添加

File: model_zoo/ernie-tiny/deploy/android/README.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    Predictor predictor = new Predictor();
+
+    // 设置模型文件和标签文件
+    String modelFile = "ernie-tiny/infer_model.pdmodel";
+    String paramsFile = "ernie-tiny/infer_model.pdiparams";
+    String vocabFile = "ernie-tiny/vocab.txt";
+    String slotLabelsFile = "ernie-tiny/slots_label.txt";
+    String intentLabelsFile = "ernie-tiny/intent_label.txt";
```
Comment (joey12300) [inline model_zoo/ernie-tiny/deploy/android/README.md:157]: 加一行added_tokens文件路径
```java
String addedTokensFile = "ernie-tiny/added_tokens.json";
```

File: model_zoo/ernie-tiny/deploy/android/README.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+
+工程内容说明：  
+```bash
+.
+├── README.md
+├── app                   # App示例代码
+├── build.gradle
+├── ernie_tiny            # ERNIE Tiny JNI & Java封装代码
+# ...
```
Comment (joey12300) [inline model_zoo/ernie-tiny/deploy/android/README.md]: 省略号的文件重要吗？例如gradle.properties, gradlew等等

Comment (DefTruth) [reply model_zoo/ernie-tiny/deploy/android/README.md]: 我加个注释

File: model_zoo/ernie-tiny/deploy/android/README.md (hunk 1 above)
Comment (DefTruth) [reply model_zoo/ernie-tiny/deploy/android/README.md]: done

File: model_zoo/ernie-tiny/deploy/android/README.md (hunk 2 above)
Comment (DefTruth) [reply model_zoo/ernie-tiny/deploy/android/README.md:157]: done

File: model_zoo/ernie-tiny/deploy/android/README.md (hunk 3 above)
Comment (DefTruth) [reply model_zoo/ernie-tiny/deploy/android/README.md]: done

**Review by joey12300 — APPROVED** [review]
LGTM

**Pull request merged**